Formula/g/goasitop.rb

```diff
@@ -0,0 +1,24 @@
+# typed: true
+# frozen_string_literal: true
+
+# goasitop homebrew formula
```

Comment:
This is not needed
```suggestion
```

Reply:
It complains about missing the description in comments?

Reply:
I'm very sure it will not, since no formula in homebrew/core has this